### DISCUSSION TITLE
Image on readme, won’t show in app

### DISCUSSION THREAD

**Discussion by pownas** [community/community#24079] [Apps, API and Webhooks]
<p>Hi!<br>
I have this image on my GitHub repo: <a href="https://raw.githubusercontent.com/pownas/ArvidsonFoto-MVC-NET5/main/docs/Anv%C3%A4ndningsfalls-modell-version1.0-2021-01-27.jpg" rel="noopener nofollow ugc">https://raw.githubusercontent.com/pownas/ArvidsonFoto-MVC-NET5/main/docs/Användningsfalls-modell-version1.0-2021-01-27.jpg</a></p>
<p>It shows fine at:</p><aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/pownas/ArvidsonFoto-MVC-NET5" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/47676607/181081835-984d1237-9ba7-4abe-96b0-96230406c7e1.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/c/3c7e683c5c120200b926edfd6940891b574ca47d_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/c/3c7e683c5c120200b926edfd6940891b574ca47d_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/3/c/3c7e683c5c120200b926edfd6940891b574ca47d.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/c/3c7e683c5c120200b926edfd6940891b574ca47d_2_10x10.png"></div>

<h3><a href="https://github.com/pownas/ArvidsonFoto-MVC-NET5" target="_blank" rel="noopener nofollow ugc">pownas/ArvidsonFoto-MVC-NET5</a></h3>

  <p><span class="label1">Ombyggnation av ArvidsonFoto med MVC och .NET5. Contribute to pownas/ArvidsonFoto-MVC-NET5 development by creating an account on GitHub.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>But won’t show up at all on the GitHub app repo page:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/5/9/59fb5801c73030cf318f8aaa04ea0c8b0d750db4.jpeg" data-download-href="/uploads/short-url/cQ0ZMgUCP8z2yW0Kv3r9uemvoUs.jpeg?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47676607/181081836-e676630a-aea6-471c-9e9b-f1fcc7f976e0.jpeg" alt="image" data-base62-sha1="cQ0ZMgUCP8z2yW0Kv3r9uemvoUs" width="230" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/9/59fb5801c73030cf318f8aaa04ea0c8b0d750db4_2_230x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/5/9/59fb5801c73030cf318f8aaa04ea0c8b0d750db4_2_345x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/5/9/59fb5801c73030cf318f8aaa04ea0c8b0d750db4_2_460x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/9/59fb5801c73030cf318f8aaa04ea0c8b0d750db4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1125×2436 185 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Can this be fixed?<br>
Or is it because I link to a image in my repo and need to change url?</p>

**Comment by jfullstackdev** [comment]
<p>is it on the README?</p>
<pre><code class="lang-auto">![ ](URL)
</code></pre>
<p>is that how you embed your image?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/b/8ba2cab54b2de735d431f061f4a23a3d1790aa71.jpeg" data-download-href="/uploads/short-url/jVh9bYAs0ItzCcooY1JHTA9IHIJ.jpeg?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181081839-cd3e734e-9804-4501-8448-4d240f20e11c.jpeg" alt="image" data-base62-sha1="jVh9bYAs0ItzCcooY1JHTA9IHIJ" width="385" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/b/8ba2cab54b2de735d431f061f4a23a3d1790aa71_2_385x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/8/b/8ba2cab54b2de735d431f061f4a23a3d1790aa71_2_577x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/b/8ba2cab54b2de735d431f061f4a23a3d1790aa71_2_770x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/b/8ba2cab54b2de735d431f061f4a23a3d1790aa71_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1207×1564 155 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I used your image and it’s being displayed here, so it’s loading</p>

**Comment by canuckjacq** [comment]
<p>Hi <a class="mention" href="/pownas">@pownas</a></p>
<p>It looks as though this is a known issue:</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/github/feedback/discussions/14" target="_blank" rel="noopener">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/362;"><img src="https://user-images.githubusercontent.com/11161530/181081841-7aab6a88-ebfe-484e-8829-c2b2453d2ace.png" class="thumbnail" width="690" height="362" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/5/55c92a5cafb55dd5ba2c844f704161bfdac81aa6_2_690x362.png, https://aws1.discourse-cdn.com/github/optimized/2X/5/55c92a5cafb55dd5ba2c844f704161bfdac81aa6_2_1035x543.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/5/55c92a5cafb55dd5ba2c844f704161bfdac81aa6.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/5/55c92a5cafb55dd5ba2c844f704161bfdac81aa6_2_10x10.png"></div>

<h3><a href="https://github.com/github/feedback/discussions/14" target="_blank" rel="noopener">Images in readme are not resizing on the mobile app · Discussion #14 ·...</a></h3>

<p><a href="https://github.com/github/feedback/discussions/14" target="_blank" rel="noopener">//github.com/github/feedback/discussions/14</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<blockquote>
<p>After some initial investigations we found that images using width and height attributes of the img HTML tag were actually not supported on mobile at the moment. We will monitor this space and make potential improvements when we have more bandwidth in the future.</p>
</blockquote>

**Comment by pownas** [comment]
<p>Yes, that is how it is applied on the readme.md. But not showing up in the mobile app at all.</p>
<p>Code in readme:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/e/ae7817da6154daca2e79ca7e1f7d78f1ad7090fa.jpeg" data-download-href="/uploads/short-url/oTqnsbIEo1tPthHblOtDzwAlMCK.jpeg?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47676607/181081843-a52e01cb-030c-4c19-b8ac-f9667299cc63.jpeg" alt="image" data-base62-sha1="oTqnsbIEo1tPthHblOtDzwAlMCK" width="690" height="318" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/e/ae7817da6154daca2e79ca7e1f7d78f1ad7090fa_2_690x318.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/a/e/ae7817da6154daca2e79ca7e1f7d78f1ad7090fa_2_1035x477.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/a/e/ae7817da6154daca2e79ca7e1f7d78f1ad7090fa_2_1380x636.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/e/ae7817da6154daca2e79ca7e1f7d78f1ad7090fa_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">2436×1125 323 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by pownas** [comment]
<p>Yes <a class="mention" href="/canuckjacq">@canuckjacq</a>.<br>
It might be the same issue as that one.<br>
But I’m not 100% sure if it’s the same error.</p>
<p>At least image ain’t showing up and it would’ve been lovely if it could work with a image in the readme on the mobile app start page for the repo.</p>

**Comment by jfullstackdev** [comment]
<p>is that the only image not loading? or other images are showing up?</p>
<p>Because there is also the possibility that it’s not loading on mobile because of the URL, as you can see, you’re URL path contains dot before the filename extension.</p>
<p><code>/Användningsfalls-modell-version1.0-2021-01-27.jpg</code></p>
<p>try changing the name of the URL without any dot, maybe just hyphen and see,</p>

**Comment by pownas** [comment, marked as answer]
<p>The trouble was that I had a ”ä” in the path of ”anvÄndningsfalls-modell” (use case-model).<br>
Replaced that with a, and then all worked.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/d/ddabc8d8d01688fe7e45aadb74a5a659670197c9.jpeg" data-download-href="/uploads/short-url/vCZzcbWsp1PpW6Pqo1WM2gOUliV.jpeg?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47676607/181081845-a0821e1b-477f-40a2-98fc-cc03b69902b8.jpeg" alt="image" data-base62-sha1="vCZzcbWsp1PpW6Pqo1WM2gOUliV" width="409" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/d/ddabc8d8d01688fe7e45aadb74a5a659670197c9_2_409x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/d/d/ddabc8d8d01688fe7e45aadb74a5a659670197c9_2_613x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/d/d/ddabc8d8d01688fe7e45aadb74a5a659670197c9_2_818x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/d/ddabc8d8d01688fe7e45aadb74a5a659670197c9_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1125×1373 152 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><p></p>
<p>Thanks <a class="mention" href="/jdevstatic">@jdevstatic</a> for pointing me in the right direction.<br>
The Swedish åäö/ÅÄÖ, is not good in the image path… :confused:</p>

**Comment by jfullstackdev** [comment]
<p>yup, but I’m not aware you are using that, so my focus is the dot,</p>
<p>that’s actually in network programming, sometimes it will be resolved, sometimes not, so if you are not sure, just use the standard encoding,</p>

**Comment by jfullstackdev** [comment]
<p>also, the dot or point, it may cause problems, we can never be sure of that, sometimes it will work, sometimes not</p>
